### PR TITLE
no busy in read

### PR DESCRIPTION
**Pull request by untzag**
busy status should not appear in the read dictionary

- [x] implementation
- [x] changelog

### CODE REVIEW

**Comment by ksunden** [comment]
@untzag ping

**Comment by untzag** [comment]
This breaks everything and I don't understand why. That's where I'm at.

**Comment by ksunden** [comment]
@untzag, the modified function is called _after_ other things are added (out is an input to it), so it is wiping everything out , just return `out` directly without doing `out = OrderedDict()`

File: yaqc_bluesky/_base.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def hints(self) -> Dict:
         return out
 
     def _read(self, out, ts) -> OrderedDict:
-        out[f"{self.name}_busy"] = {"value": self.yaq_client.busy(), "timestamp": ts}
+        out = OrderedDict()
```
Comment (ksunden) [inline yaqc_bluesky/_base.py]:
```suggestion
```

**Comment by untzag** [comment]
thanks @ksunden

**Comment by ksunden** [comment]
Okay, there were some places that it was used internally that needed to be removed as well....

**Pull request merged**